src/app/screens/connectors/NewWallet/index.tsx

```diff
@@ -136,21 +146,66 @@ export default function NewWallet() {
       ) : (
         <>
           <div className="mt-6 dark:text-white">
-            <strong>We host a lightning wallet for you!</strong>
+            <strong>Create a getAlby.com account</strong>
             <br />
-            ...but remember: not your keys, not your coins.
+            ...and let us host a lightning wallet for you!
           </div>
 
           <div className="mt-6">
             <TextField
               id="email"
               label="Email Address"
               type="email"
+              required
               onChange={(e) => {
                 setEmail(e.target.value.trim());
               }}
             />
           </div>
+          <div className="mt-6">
+            <TextField
+              id="password"
+              label="Password"
+              type="password"
+              minLength={6}
+              pattern=".{6,}"
+              title="at least 6 characters"
+              required
+              onChange={(e) => {
+                setPassword(e.target.value.trim());
+              }}
+            />
+          </div>
+          <div className="mt-6">
+            <p className="text-gray-500 dark:text-gray-400">
+              Your Alby account also comes with an optional{" "}
+              <a
+                href="https://lightningaddress.com/"
+                target="_blank"
+                rel="noreferrer"
+              >
+                Lightning Address
+              </a>
+              . This is a simple way for anyone to send you Bitcoin on the
+              Lightning Network. (
+              <a
+                href="https://lightningaddress.com/"
+                target="_blank"
+                rel="noreferrer"
+              >
+                learn more
+              </a>
+              )
+            </p>
+            <TextField
+              id="lnAddress"
+              label="Choose your Lightning Address (optional)"
+              type="text"
+              onChange={(e) => {
+                setLnAddress(e.target.value.trim().split("@")[0]); // in case somebody enters a full address we simple remove the domain
```

Comment:
Because an Alby address will always have the `@getAlby.com` domain? Not too familiar with this yet.  
If yes, can a placeholder for the input help to avoid confusion and set expectations?
If an address is already taken it will lead to "Failed to create a new wallet" further up, yes?

Reply:
because people might enter @getalby.com or something. 

and yes, something as we have on the web app would be good, but I did not know how :) 
<img width="498" alt="image" src="https://user-images.githubusercontent.com/318/160789709-c28cc2c9-d915-4f6c-a103-40e37ab61bfd.png">

if the address is taken, then yes, it would fail with that alert (including the errors as JSON) - there are no good error messages so far.